tests/regression/test_2dcohomology.py

```diff
@@ -94,8 +94,8 @@ def test_betti1(space, mesh):
     L = assemble((inner(sigma, tau) - inner(u, rot(tau)) + inner(rot(sigma), v)
                   + inner(div(u), div(v))) * dx)
 
-    bc0 = DirichletBC(W.sub(0), Constant(0.0), 9)
-    bc1 = DirichletBC(W.sub(1), Constant((0.0, 0.0)), 9)
```

Comment:
Why this change?

Reply:
Nothing wrong here, just had to deal with similar bcs on tests, where I replaced `zero(V.ufl_element().value_shape)` with `0`. Prescribing a `Constant` will require to generate code, so this gives us a tiny speedup